ui/admin/app/controllers/scopes/scope/workers/worker/tags.js

```diff
@@ -70,4 +113,16 @@ export default class ScopesScopeWorkersWorkerTagsController extends Controller {
     this.modalTag = apiTag;
     this.removalConfirmation = null;
   }
+
+  /**
+   * Opens the edit tag modal.
+   * @param {object} apiTag
+   */
+  @action
+  toggleEditModal(apiTag) {
+    this.editModal = !this.editModal;
+    this.modalTag = apiTag;
+    this.editKey = apiTag?.key;
+    this.editValue = apiTag?.value;
```

Comment:
I'm not sure I understand why you need `editKey` and `editValue` if you're already saving the apiTag to the `modalTag`, couldn't you have just used only `modalTag` for the edited values?

Since you already have access to the data row from the table you always know what the original was

Reply:
Discussed this with Zed and we do not have access to the row data so we are leaving how it is for now.